solidity/dashboard/src/components/Rewards.jsx

```diff
@@ -197,7 +197,7 @@ const RewardsComponent = ({ withdrawRewardAction }) => {
                 tooltipText={amountTooltipText(totalRewardsBalance)}
               />
               <div className="ml-1 self-center">
-                <SpeechBubbleTooltip text="The total balance reflects the total Available and Acummulating rewards. Available rewards are ready to be withdrawn. Acummulating rewards become available after a signing group expires." />
+                <SpeechBubbleTooltip text="The total balance reflects the total Available and accumulating rewards. Available rewards are ready to be withdrawn. Acummulating rewards become available after a signing group expires." />
```

Comment:
Shouldn't `Available` also be updated?

```suggestion
                <SpeechBubbleTooltip text="The total balance reflects the total available and accumulating rewards. Available rewards are ready to be withdrawn. Acummulating rewards become available after a signing group expires." />
```

Reply:
Thank you, will fix

Reply:
There's actually also another instance of acummulating->accumulating in the next sentence 🙆

Reply:
@nkuba I think that we should keep first letter as capital in both Available and Accumulating, because those are the names of the rewards. What do you think?

![image](https://user-images.githubusercontent.com/40306834/100734691-56f45780-33d0-11eb-8f99-0196f4323eee.png)

Reply:
We're describing the concepts in this sentence, rather than calling them out as proper nouns or special items. Mostly it's bikeshedding, tbh, but I do strongly feel the right move is to lowercase them both.

Reply:
@Shadowfiend I see. I will keep the lowercase in both words then. Thanks 👍